### PR TITLE
DM-6614: Add Kron configuration info to metadata

### PR DESCRIPTION
**Pull request by TallJimbo**

### CODE REVIEW

File: src/KronPhotometry.cc [hunk 1]
```diff
@@ -432,6 +436,15 @@ KronFluxAlgorithm::KronFluxAlgorithm(KronFluxControl const & ctrl, std::string c
...
     _flagHandler = meas::base::FlagHandler::addFields(schema, name, flagDefs.begin(), flagDefs.end());
+    metadata.add(name + "_fixed", ctrl.fixed);
+    metadata.add(name + "_nSigmaForRadius", ctrl.nSigmaForRadius);
+    metadata.add(name + "_nIterForRadius", ctrl.nIterForRadius);
+    metadata.add(name + "_nRadiusForFlux", ctrl.nRadiusForFlux);
+    metadata.add(name + "_maxSincRadius", ctrl.maxSincRadius);
+    metadata.add(name + "_minimumRadius", ctrl.minimumRadius);
+    metadata.add(name + "_enforceMinimumRadius", ctrl.enforceMinimumRadius);
+    metadata.add(name + "_useFootprintRadius", ctrl.useFootprintRadius);
+    metadata.add(name + "_smoothingSigma", ctrl.smoothingSigma);
```
Comment (TallJimbo) [inline src/KronPhotometry.cc]: I don't think we need all of this; the metadata should include the information needed to interpret the results, but it doesn't need to include all of the configuration (and given that it doesn't for other algorithms I don't we want to selectively do that here).  I'd recommend just setting "nRadiusForFlux".  @PaulPrice, do you concur?

File: tests/Kron.py [hunk 2]
```diff
@@ -173,6 +151,44 @@ def tearDown(self):
...
+        refCat = afwTable.SourceCatalog(source.table)
+        refCat.append(source)
+        schema = afwTable.SourceTable.makeMinimalSchema()
+        algMeta = PropertyList()
+        task = measBase.ForcedMeasurementTask(schema, config=msConfig, algMetadata=algMeta)
+        measCat = task.generateMeasCat(exposure, refCat, refWcs)
+        task.attachTransformedFootprints(measCat, refCat, exposure, refWcs)
+        task.run(measCat, exposure, refCat, refWcs)
+        self.checkAlgMeta(algMeta)
+        return measCat[0]
```
Comment (TallJimbo) [inline tests/Kron.py]: If these methods don't use `self`, they should probably be static.

File: src/KronPhotometry.cc (hunk 1 above)
Comment (ktlim) [inline src/KronPhotometry.cc]: Configuration is already persisted separately (and ought to be retrievable by anyone accessing result datasets just as easily as processing metadata), so I'd generally avoid putting any configuration information into the processing metadata.

Comment (RobertLuptonTheGood) [inline src/KronPhotometry.cc]: I agree with all of you...  No configuration, except nRadiusForFlux is something you need to understand the measurements (e.g. plotting the aperture used).

Comment (AstroVPK) [inline src/KronPhotometry.cc]: Ok, I'm taking out all the metadat other than nRadiusForFlux.

File: tests/Kron.py (hunk 2 above)
Comment (AstroVPK) [inline tests/Kron.py]: Ok, made the change.

**Pull request merged**